include/robot_interfaces/robot_logger.hpp

```diff
@@ -94,9 +96,19 @@ class RobotLogger
         thread_->join();
     }
 
+    /*
+    * get_header() is to get the title of the log file, describing all the information
+    * that will be logged in it.
+    */
+
```

Comment:
Please use the doxygen syntax for those comments (see [here](https://github.com/open-dynamic-robot-initiative/robot_interfaces/blob/7be66054198c70a47e303063417aeae619b76ecc/include/robot_interfaces/monitored_robot_driver.hpp#L88) for an example).
Also please omit the empty line between comment block and function.